tools/test-proxy/Azure.Sdk.Tools.TestProxy.Tests/TransformTests.cs

```diff
@@ -27,7 +27,10 @@ public async Task ConditionalTransformAppliesForRegex()
             var untransformedEntry = TestHelpers.LoadRecordSession(targetFile).Session.Entries[1];
 
             // start playback
-            playbackContext.Request.Headers["x-recording-file"] = targetFile;
+            var body = "{\"x-recording-file\":\"" + targetFile + "\"}";
```

Comment:
Can we add a test case with non-ascii chars so we make sure everything works as expected?